index.bs

```diff
@@ -248,10 +248,11 @@ To <dfn abstract-op lt="set-site">set the `Sec-Fetch-Site` header</dfn> for a [=
 
         1.  If |url| is [=same origin=] with |r|'s [=request/origin=], [=iteration/continue=].
 
-        2.  If |r|'s [=request/origin=]'s [=origin/scheme=] is not the same as |url|'s
-            [=url/scheme=], or if |r|'s [=request/origin=]'s [=registrable domain=] is not the same
-            as |url|'s [=registrable domain=], set |header|'s value to `cross-site` and
-            [=iteration/break=].
+        2.  Set |header|'s value to `cross-site` and break if any of the following are true:
+
+            *  |r|'s [=request/origin=] is an [=opaque origin=].
+            *  |r|'s [=request/origin=]'s [=origin/scheme=] is not the same as |url|'s
```

Comment:
need to add url/scheme at the end here?

Reply:
Yup, that was dumb. Fixed in https://github.com/w3c/webappsec-fetch-metadata/pull/42/commits/f3cd5c10ff4c627e2bc8de4447a59a9ac7b2bd1e.